run.py

```diff
@@ -255,6 +257,14 @@ def _bulk_find(resource, args):
             links['next'] = { 'page': (page+1) }
             links['last'] = { 'page': pages }
 
+        for item in items:
+            for k, v in item.items():
+                if isinstance(v, datetime):
+                    formatted_date = '{:%m/%d/%Y}'.format(v)
```

Comment:
Dates should to be formatted according to whatever the `DATE_FORMAT` config setting is (defaults to RFC1123 apparently). 

http://python-eve.org/config.html#global-configuration has more info